apps/asap-server/src/entities/user.ts

```diff
@@ -30,7 +30,7 @@ export const userUpdateSchema = Joi.object({
   lastName: Joi.string().allow(''),
   jobTitle: Joi.string().allow(''),
   degree: Joi.string()
-    .allow('BA', 'BSc', 'MSc', 'PhD', 'MD', 'PhD, MD')
+    .valid('BA', 'BSc', 'MSc', 'PhD', 'MD', 'PhD, MD')
```

Comment:
It be nice if we could take these values from the model. Ive been turning the enums into arrays or sets and then dynamically generating a type from it for the model and using the values where appropirate